README.md

```diff
@@ -2,9 +2,24 @@
 The ATAT Web API repo contains an implementation of the internal application programming interface that supports the [ATAT Web UI](https://github.com/dod-ccpo/atat-web-ui). This API will eventually invoke the [ATAT CSP Orchestration](https://github.com/dod-ccpo/atat-csp-orchestration) layer.
 
 ## Deploying to AWS
-The document contains helpful information for installing and deploying the implementation of the _ATAT Portfolio Drafts API_ to AWS.
+The document contains helpful information for installing and deploying the implementation of the _ATAT Portfolio Drafts API_ to AWS. [AWS Cloud Development Kit](https://aws.amazon.com/cdk/) (AWS CDK) framework is used for deployment. The `cdk.json` provides configuration to the AWS CDK Toolkit.
 
-The `cdk.json` file tells the [AWS CDK Toolkit](https://docs.aws.amazon.com/cdk/latest/guide/cli.html) how to execute your app.
+### Install Prerequisites
+These software packages are required.
+
+| Package | Command | Notes |
+| ----------- | ----------- |----------- |
+| Node Version Manager | `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh \| bash` | Then restart your shell. Details available in the [nvm README](https://github.com/nvm-sh/nvm#installing-and-updating). |
+| Node.js | `nvm install` | Node.js version is specified in file `./.nvmrc` |
+| TypeScript | `npm install -g typescript` | [Documentation](https://www.typescriptlang.org/docs/) |
+| AWS CDK Toolkit | `npm install -g aws-cdk` | [Developer Guide](https://docs.aws.amazon.com/cdk/latest/guide/cli.html) |
+
+### Install JavaScript Dependencies 
+```
+npm ci
+npm run bootstrap
```

Comment:
Need to install first otherwise no lerna for bootstrap

Reply:
This is assuming running commands from root.

Reply:
I think that is a perfectly fair assumption